cmd/common-main.go

```diff
@@ -98,7 +98,7 @@ func handleCommonCmdArgs(ctx *cli.Context) {
 func handleCommonEnvVars() {
 	// Start profiler if env is set.
 	if profiler := os.Getenv("_MINIO_PROFILER"); profiler != "" {
-		globalProfiler = startProfiler(profiler)
+		globalProfiler = startProfiler(profiler, "")
```

Comment:
We should have been returning an error when profiler env variable is set to an unsupported value - probably we can do this now.

At startup when checking the environment variable, lets say its mistyped as `cpuu` instead of `cpu` it will be quite unclear why profiling is not happening as the server does not log/warn/etc, but silently does not do any profiling.

Reply:
> We should have been returning an error when profiler env variable is set to an unsupported value - probably we can do this now.
> 
> At startup when checking the environment variable, lets say its mistyped as `cpuu` instead of `cpu` it will be quite unclear why profiling is not happening as the server does not log/warn/etc, but silently does not do any profiling.

Fixed